torch/csrc/jit/passes/decompose_ops.cpp

```diff
@@ -108,7 +108,8 @@ bool DecomposeOps(Block* block, script::CompilationUnit& decompose_funcs) {
       decomposed = true;
       WithInsertPoint guard(*it);
       std::shared_ptr<Graph> d_graph =
-          decompose_funcs.get_function("addmm").graph();
+          dynamic_cast<FunctionImpl&>(decompose_funcs.get_function("addmm"))
```

Comment:
Maybe we should have .graph() but allow it to return a nullptr if it is a built in, it would clean up all these dynamic casts in places where we know there should be a graph representation.